notebooks/Notebook1.md

```diff
@@ -2,33 +2,71 @@
+So I may borrow some of the Prezi features, but hope to offer more beyond their structure.
+I looked at interactive videos as well and their capabilities are more in line with what I want to create, but I don't want my presentations to need a video base and want them to be more aimed toward an output that someone could use to present rather than something someone would explore on their own.
+So again, I could definitely use some of those features, but since it's a slightly different domain, I want to tailor mine more toward presentation capabilities.
+I looked a bit at React and Swagger which were a couple suggestions in class, but I was unable to see how they would apply to my project.
+
+I've thought a bit about what my intermediate representation will be and am currently thinking it will be a list of HTML components with a separate CSS output for styling for the elements and a list of JavaScript instructions for each transition.
+This is definitely likely to change though once I actually start implementing it.
+
+I also agreed that it is very helpful to have specific ideas prepared for class so I can get helpful feedback, so I considered how I will condense my main ideas here to highlight what I most need help with and also give a clear description of where I am so far.
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+The main things I worked on this week were creating an [example output](<https://hmc-cs111-spring2023.github.io/project-code-occamkg/work%20files/basicPres0/example%20output>) and designing an [example syntax](<https://github.com/hmc-cs111-spring2023/project-code-occamkg/tree/master/work%20files/basicPres0/example%20syntax>).
+I spent most of my time on creating the output because I also needed to refamiliarize myself with web language programming.
+
+I got a lot of the basics down in terms of what I will need for every output.
+For example, in the JS file, I worked on how to translate the keyboard input into navigating the presentation timeline, which is a fundamental part of presentations.
+Similarly, in the style, I figured out how to make a content area that can be defined by the aspect ratio and fill the screen no matter its shape which is also essential for presentations that can function consistently.
+
+I kept the example pretty basic (only text elements so far), but I wanted to add capabilities that exemplify how these web languages can expand presentation capabilities, so I made sure to include transitions that affect style of different elements (I changed size and opacity of text).
```

Comment:
I think adding simple things like images won't be too much harder than text because they can have all the same features that texts do(like the same transitions).